superbench/executor/executor.py

```diff
@@ -106,17 +110,47 @@ def __exec_benchmark(self, context, log_suffix):
                     logger.info('Executor succeeded in %s.', log_suffix)
                 else:
                     logger.error('Executor failed in %s.', log_suffix)
+                return benchmark.result
             else:
                 logger.error('Executor failed in %s, invalid context.', log_suffix)
-        except Exception:
+        except Exception as e:
+            logger.error(e)
             logger.error('Executor failed in %s.', log_suffix)
+        return None
+
+    def __get_benchmark_dir(self, benchmark_name, create=False):
+        """Get output directory for benchmark.
+
+        Args:
+            benchmark_name (str): Benchmark name.
+            create (bool): Create the directory or not.
+
+        Return:
+            str: Benchmark output directory.
+        """
+        benchmark_output_dir = Path(self._output_dir, 'benchmarks', benchmark_name)
+        if create:
+            benchmark_output_dir.mkdir(mode=0o755, parents=True, exist_ok=True)
```

Comment:
Do we need to empty the dir?

Reply:
nope? if the same output dir is used, later output will rewrite previous one

Reply:
added directory rotation and keep it empty